### ISSUE TITLE
Should provide other ways to specify the scaling target beside name.

### ISSUE THREAD

**Issue by MrHohn** [kubernetes-incubator/cluster-proportional-autoscaler#9]
As the scaling target's name could be changed, autoscaler should provide other ways for user to specify what the target is. One candidate would be `label selector`.

Take kube-dns as an example, we could use label `k8s-app: kube-dns` to select the target ReplicationController/Deployment, and don't need to restart autoscaler to change the input argument everytime when the target name is changed.

**Comment by vijaygos** [comment]
I think this is a much-needed feature. We have a cluster with multiple nodepools and we would like to scale independently for each of these node pools. Having a label selector to filter the number of nodes allows for scaling in such scenarios.

**Comment by MrHohn** [comment]
@vijaygos For you use case, is that https://github.com/kubernetes-incubator/cluster-proportional-autoscaler/pull/55 (filter node based on labels)?

**Comment by vijaygos** [comment]
Yes. This would work. I just realized that the [change ](https://github.com/kubernetes-incubator/cluster-proportional-autoscaler/commit/79563c6b2f352f04046e12772864e5c24ef48d77) went in recently. 
Thanks for adding that. Much appreciated.

**Comment by MrHohn** [comment]
@vijaygos In case you don't want to build it from head, I will publish a new image includes that change very soon.

**Comment by vijaygos** [comment]
Would it be possible for you to comment on your timeline for a formal release?

**Comment by MrHohn** [comment]
@vijaygos I just did --- images below should be available now:
- k8s.gcr.io/cluster-proportional-autoscaler-amd64:1.5.0
- k8s.gcr.io/cluster-proportional-autoscaler-arm:1.5.0
- k8s.gcr.io/cluster-proportional-autoscaler-arm6464:1.5.0
- k8s.gcr.io/cluster-proportional-autoscaler-ppc64le:1.5.0

**Comment by vijaygos** [comment]
Awesome! Thanks@MrHohn for the quick turnaround.
